SpatialGDK/Source/SpatialGDK/Private/Interop/RPCExecutor.cpp

```diff
@@ -10,7 +11,7 @@
 
 namespace SpatialGDK
 {
-RPCExecutor::RPCExecutor(class USpatialNetDriver* InNetDriver)
+RPCExecutor::RPCExecutor(USpatialNetDriver* InNetDriver)
```

Comment:
Uber Nit: It might be preferential to pass in the event tracer? Might not be the case now but it might make testing easier in some far off universe.

Reply:
yeah that makes sense!